test/rails_app/config/application.rb

```diff
@@ -46,7 +46,7 @@ class Application < Rails::Application
     end
 
     # Remove this check once Rails 5.0 support is removed.
-    if Devise::Test.rails52_and_up?
+    if Devise::Test.rails52_and_up? && !Devise::Test.rails6?
```

Comment:
Probably something like `# Remove the first check once Rails 5.0 support is removed.` now? Do you know if Rails 6 is always going to require this? (It might make sense to update the docs to consider it too.)

Reply:
We'll still need the `rails6?` check once we remove support for Rails 5 because here we need to set this config for Rails 5.2. When Rails 6 is the only supported version, then we can remove this code altogether.